joblib/parallel.py

```diff
@@ -300,6 +371,14 @@ class Parallel(Logger):
             - finally, you can register backends by calling
               register_parallel_backend. This will allow you to implement
               a backend of your liking.
```

Comment:
I think that we need to revisit the docstring and the arguments to deprecate "backend" and replace it by the mechanism of backend specification that you implemented and the default backend.

What do people think?